<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CourseUI</class>
 <widget class="QWidget" name="CourseUI">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: #fff;
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
	background-color: #fff;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="inforFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>531</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background-color: rgba(230, 230, 230, 255);
	border-radius: 15px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Author:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Day modified:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>190</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Rating:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>230</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Description:</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="descriptionTxtBox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>230</y>
       <width>321</width>
       <height>181</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLabel" name="dayModifyLbl">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>150</y>
       <width>321</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Day modified:</string>
     </property>
    </widget>
    <widget class="QLabel" name="ratingLbl">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>190</y>
       <width>321</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Rating:</string>
     </property>
    </widget>
    <widget class="QLabel" name="authorLbl">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>110</y>
       <width>321</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Author:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="registerBtn">
     <property name="geometry">
      <rect>
       <x>207</x>
       <y>458</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton {
   background-color: rgb(97, 31, 31);
   border-radius: 8px;  
   color: #fff;
   font-family: Inter,sans-serif;
   font-size: 1.2rem;
   font-weight: 700;
   line-height: 1;
   margin: 0;
   outline: none;
   padding: 1rem 1.2rem;
   text-align: center;
   text-decoration: none;
   white-space: nowrap;
   border: 0;
 }
QPushButton:hover:!pressed
{
  background-color: rgb(100, 30, 30);
}
 QPushButton:hover {
     background-color: rgb(147, 81, 81);
 }</string>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
    <widget class="QLabel" name="courseName">
     <property name="geometry">
      <rect>
       <x>43</x>
       <y>30</y>
       <width>451</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>COURSES NAME</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>33</x>
       <y>467</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> QPushButton {
   background-color: rgb(97, 31, 31);
   border-radius: 8px;  
   color: #fff;
   font-family: Inter,sans-serif;
   font-size: 1.2rem;
   font-weight: 700;
   line-height: 1;
   margin: 0;
   outline: none;
   padding: 1rem 1.2rem;
   text-align: center;
   text-decoration: none;
   white-space: nowrap;
   border: 0;
 }
QPushButton:hover:!pressed
{
  background-color: rgb(100, 30, 30);
}
 QPushButton:hover {
     background-color: rgb(147, 81, 81);
 }</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>label_4</zorder>
    <zorder>label_5</zorder>
    <zorder>descriptionTxtBox</zorder>
    <zorder>dayModifyLbl</zorder>
    <zorder>ratingLbl</zorder>
    <zorder>authorLbl</zorder>
    <zorder>courseName</zorder>
    <zorder>registerBtn</zorder>
    <zorder>pushButton</zorder>
   </widget>
   <widget class="QFrame" name="testList">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>261</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background-color: rgba(255, 255, 255, 255);
	border-radius: 15px;
}

QTextEdit {
	background-color: rgba(0, 0, 0, 0);
	border: 0px;
}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
